From 8e6a2101624053c79ea39fe4aed435a3bd5a6f3f Mon Sep 17 00:00:00 2001
From: Murat Sezgin <msezgin@codeaurora.org>
Date: Fri, 24 Apr 2020 10:59:16 -0700
Subject: [PATCH 233/281] net: Export dev_shutdown()

This API is needed by interface drivers to ensure qdiscs are destroyed
before the NSS interfaces are destroyed. Since Linux destroys qdiscs
only at unregister_netdevice(), it is too late for destroying any NSS
qdiscs that may be configured on the interface.

Change-Id: I66a6ec5b5393f9848ff1202e2753c738786c515f
Signed-off-by: Shyam Sunder <ssunde@codeaurora.org>
---
 net/sched/sch_generic.c | 1 +
 1 file changed, 1 insertion(+)

--- a/net/sched/sch_generic.c
+++ b/net/sched/sch_generic.c
@@ -1494,6 +1494,7 @@ void dev_shutdown(struct net_device *dev
 
 	WARN_ON(timer_pending(&dev->watchdog_timer));
 }
+EXPORT_SYMBOL(dev_shutdown);
 
 /**
  * psched_ratecfg_precompute__() - Pre-compute values for reciprocal division
